### PR TITLE
LPS-148953 [CP 2.0] Adding logic to Activation form for liferay staff

### PR DESCRIPTION
**Pull request by igordmf**

### CODE REVIEW

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/context/index.js [hunk 1]
```diff
@@ -96,9 +96,14 @@ const AppContextProvider = ({assetsPath, children, page}) => {
 						({name}) => name === ROLE_TYPES.admin.key
 					);
 
+				const isStaff = data.userAccount?.roleBriefs?.some(
+					(role) => role.name === 'Administrator'
```
Comment (iagochaves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/context/index.js:100]: Import & use the ROLE_TYPES variable here 
```suggestion
					(role) => role.name === ROLE_TYPES.admin.name
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/context/index.js [hunk 2]
```diff
@@ -63,9 +63,14 @@ const AppContextProvider = ({assetsPath, children}) => {
 						({name}) => name === ROLE_TYPES.admin.key
 					);
 
+				const isStaff = data.userAccount?.roleBriefs?.some(
+					(role) => role.name === 'Administrator'
```
Comment (iagochaves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/context/index.js:67]:
```suggestion
					(role) => role.name === ROLE_TYPES.admin.name
```

**Review by iagochaves — CHANGES_REQUESTED** [review]
Nice

**Pull request closed without merging**